packages/data-stores/src/plans/selectors.ts

```diff
@@ -54,8 +71,14 @@ export const getPlansPaths = ( state: State ): string[] => {
 
 export const getPrices = ( state: State, _: string ): PricesMap => state.prices;
 
+export const getDiscounts = ( state: State ): DiscountsMap => {
+	// call getPrices to trigger the resolver in case someone only wants to get the discounts
+	select( STORE_KEY ).getPrices( '' );
+	return state.discounts;
+};
+
```

Comment:
It seems that if `getDiscounts()` is called before `getPrices()`,  in that case the prices that are fetched are always assuming `locale = 'en'` (because of the call on line 76).

We could either add `locale` as an argument of the `getDiscounts()` selector, or maybe compute and return the discounts information directly inside the `getPrices()` selector